copilot_proxy/app.py

```diff
@@ -35,6 +35,7 @@ async def fauxpilot_handler(request: Request, exc: FauxPilotException):
     )
 
 @app.post("/v1/engines/codegen/completions")
+@app.post("/v1/engines/copilot-codex/completions")
```

Comment:
Basically, Fauxpilot is an open source -based inferencing server for running on a public model like Codegen. So I'm curious about this PR. First of all, what is the purpose you want to create a fake Copilot certification? So what do you want to do with this PR? I want you to kindly explain this to the body of PR or Commit.

Reply:
After reading the link to https://github.com/fauxpilot/fauxpilot/discussions/72 it actually makes sense to approve this MR.